test.rb

```diff
@@ -29,4 +29,8 @@ def style
       a
   end
 
+  def test
+    a, b = 1,2;
```

Comment:
Useless assignment to variable - `a`. Use `_` or `_a` as a variable name to indicate that it won't be used.<br>Do not use parallel assignment.<br>Useless assignment to variable - `b`. Use `_` or `_b` as a variable name to indicate that it won't be used.<br>Space missing after comma.<br>Do not use semicolons to terminate expressions.